in_toto/models/mock_link.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+"""
+<Program Name>
+  mock_link.py
+
+<Author>
+  Shikher Verma <root@shikherverma.com>
+
+<Started>
+  June, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Provides a class for mock link metadata which is information gathered when a
+  step of the supply chain is run as mock.
```

Comment:
Do you mind adding a little note that summarizes the discussion we had in the PR (why we need a separate MockLink class, alternatives, ...)